src/operations/secrets.md

```diff
@@ -0,0 +1,86 @@
+ 
+1.  Send an email to operations@tomesh.net with the following information:
+	- Your Name
+	- GitHub Handle
+	- Detailed descripition of your use case or requirement.
+
+2.  A member of our Project Operations team will invite you to Bitwarden.
+	- Bitwarden will send you an email from tomeshnet@gmail.com
+	- Please make sure to whitelist the email address.
+  
+3.  Use the link in the invitation to create an account.
+
+4.  After registration, Bitwarden requires us to confirm your account. Upon confirmation, you will have access to the organization.
+
+We recommend that you setup [two-factor authentication](https://bitwarden.com/help/article/setup-two-step-login/) on your account to increase the security of your account. 
+
+
+# Access Control
+When storing credentials, you must store them within the [Toronto Community Network organization](https://tomesh.infectedroot.com/#/organizations/e4573938-547f-4157-a977-b28fc15bcec0/vault). Bitwarden organizes secrets in groups called [Collections](https://bitwarden.com/help/article/collections/) and within those [Collections](https://bitwarden.com/help/article/collections/) are four [User Types]( https://bitwarden.com/help/article/user-types-access-control/#user-types): _User_, _Manager_, _Admin_, and _Owner_.
+
+In order to maintain integrity of the Toronto Community Network operations, most users will be granted only  User or Manager access. The Project Operations team will determine your access based on your use case or requirements.
+
+
+# Storing Secrets
+All secrets must be attached to a [Collection](https://bitwarden.com/help/article/collections/) to faciliate access control. Please be sure to attach the secret to an existing [Toronto Community Network Work Group](https://github.com/orgs/tomeshnet/teams/toronto-community-network/teams). These collection names start with "_OU".
+
+Additional Collections may be created or utilized to classify the secret such as _Social_, _Service_ _Accounts_, _Website_, and _more_.
+
+When naming your credential, please use the following convention: _System/Function - Identifer_.
+
+> **For example:**
+> - A public SSH key may be named _"Public Key - Contributor A"_
+> - A GitHub deployment key may be named _"GitHub - Deploy Key (docs.tomesh.net)"_
+> - A shared login social media may be named _"Twitter - tomeshnet"_
+
+When determining a password or passphrase, please use strong entropy to prevent unauthoized access. For more information, please see our [Recommended Best Practices](#recommended-best-practices) below.
+
+All public keys and private keys must be saved as _Secured Notes_.
+
+Please be mindful that secrets stored in this system can be accessed by any authorized contributor in the Toronto Community Network. While your connection to the system is secured via SSL, the infrastructure it is hosted on is not PCI-DSS compliant. Please refrain from using the _Card_ and _Identity_ item types for storing any personal information.
```

Comment:
```suggestion
Please be mindful that secrets stored in this system can be accessed by any authorized contributor in the Toronto Community Network. While your connection to the system is secured via TLS, the secrets you choose to store in the group can be seen by others. Please refrain from using the system for storing any personal information.
```

Is this more clear? I feel like bringing up PCI-DSS complicates things, and is not a very well known acronym.